Tests/NIOHTTPCompressionTests/HTTPRequestCompressorTest.swift

```diff
@@ -0,0 +1,376 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import CNIOExtrasZlib
+import NIO
+import NIOHTTP1
+@testable import NIOHTTPCompression
+
+struct PromiseArray {
```

Comment:
sorry, that's a nit. Would you mind moving all this to the bottom? Just so that the uni test class is up top in the file. That makes it much easier to find the "run this test suite" diamond button in Xcode :)

Reply:
done